### PR TITLE
Update installing-istio.md

### PR DESCRIPTION
**Pull request by xtreme-sameer-vohra**
## Proposed Changes
Remove line with link about managed instio installation as that just links back to the install guide and doesn't provide any specific guidance.

### CODE REVIEW

**Comment by xtreme-sameer-vohra** [comment]
@googlebot I fixed it.

File: docs/install/installing-istio.md [hunk 1]
```diff
@@ -9,9 +9,7 @@ with Knative.
 
 If your cloud platform offers a managed Istio installation, we recommend
 installing Istio that way, unless you need the ability to customize your
```
Comment (abrennan89) [inline docs/install/installing-istio.md:11]:
```suggestion
installing Istio that way, unless you need to customize your
```

**Comment by xtreme-sameer-vohra** [comment]
squashed and re-pushed to fix CLA issue

**Pull request merged**